app/models/attorney_case_reviews/attorney_case_review.rb

```
@@ -8,17 +8,19 @@ class AttorneyCaseReview < ActiveRecord::Base
   # task ID is vacols_id concatenated with the date assigned
   validates :task_id, format: { with: /\A[0-9]+-[0-9]{4}-[0-9]{2}-[0-9]{2}\Z/i }
 
-  EXCEPTIONS = [QueueRepository::ReassignCaseToJudgeError, VacolsHelper::MissingRequiredFieldError].freeze
+  EXCEPTIONS = [QueueRepository::ReassignCaseToJudgeError,
+                VacolsHelper::MissingRequiredFieldError,
+                ActiveRecord::RecordInvalid].freeze
 
   class << self
     attr_writer :repository
 
     def complete!(params)
       transaction do
-        # Save to the Caseflow DB first to ensure required fields are present
-        record = create(params)
-        return unless record.valid?
```

awesome, thank you for the care! i thought it would be better to throw & catch instead of failing silently :)